packages/design-system-v2/package.json

```diff
@@ -60,6 +61,7 @@
     "prettier": "^2.8.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-stately": "^3.20.0",
```

Comment:
I thought we made the decision to only install the sub packages we need. Did you mean to install something else here?

Reply:
Fixed this

Reply:
Hmm I don't see the fix?

Reply:
packages/design-system/src/Checkbox/Checkbox.tsx -> line number 2